@xen-orchestra/backups/_VmBackup.js

```diff
@@ -115,7 +115,7 @@ exports.VmBackup = class VmBackup {
 
     const settings = this._settings
 
-    assert.notStrictEqual(settings.offlineBackup, settings.snapshotRetention !== 0)
+    assert(!settings.offlineBackup || settings.snapshotRetention === 0)
```

Comment:
`||` ?

Reply:
If offline backup is enable, the rooling snapshot should be diabled.

Reply:
Shouldn't we throw only if offline backup is disabled *and* snapshot retention is 0?

Reply:
This case is fine, a backup can run when offline backup and rolling snapshot are disable.